### PR TITLE
BP-865 #comment - eliminated redundant concepts including SecuritiesO…

### PR DESCRIPTION
**Pull request by ElisaKendall**
…ffering from provisional issuance ontologies

Signed-off-by: Elisa Kendall <ekendall@thematix.com>

## Description

This resolution eliminates a number of redundant concepts from the [IssuanceProcessTerms](https://spec.edmcouncil.org/fibo/ontology/BP/SecuritiesIssuance/IssuanceProcessTerms/) and related provisional ontologies.

Fixes: #865

## Checklist:

- [x] I'm familiar with the [FIBO developer quide](../CONTRIBUTING.md#contributing-to-the-fibo-code). My contribution meets all the requirements described there.
- [x] My contribution follows the [principles of best practices for FIBO](../ONTOLOGY_GUIDE.md).

### CODE REVIEW

File: BP/SecuritiesIssuance/IssuanceProcessTerms.rdf [hunk 1]
```diff
@@ -146,7 +146,7 @@
 		<rdfs:subClassOf rdf:resource="&fibo-fnd-dt-fd;RegularSchedule"/>
 		<rdfs:subClassOf>
 			<owl:Restriction>
-				<owl:onProperty rdf:resource="&fibo-bp-iss-trm;expression"/>
+				<owl:onProperty rdf:resource="&fibo-fnd-acc-cur;hasMonetaryAmount"/>
```
Comment (rivettp) [inline BP/SecuritiesIssuance/IssuanceProcessTerms.rdf]: This does not seem right - the domain is MonetaryAmount not MonetaryMeasure.

Comment (ElisaKendall) [reply BP/SecuritiesIssuance/IssuanceProcessTerms.rdf]: I think you are right, but I also think the restriction is unnecessary.  There is an inherited restriction for the payment which includes an amount.  That can be set via an algorithm, on an ad hoc basis, or as a regularly scheduled set of payments.  The question here is whether or not the class itself is needed at all.  It seems redundant with its parent class - whose definition is a schedule for partial payments.  I'm inclined to delete the whole class, including the restriction.  What do you think?

Comment (ElisaKendall) [reply BP/SecuritiesIssuance/IssuanceProcessTerms.rdf]: Nothing references it, FYI

Comment (rivettp) [reply BP/SecuritiesIssuance/IssuanceProcessTerms.rdf]: The class doesn't seem useful to me, but Ill approve the PR as it is, now you've made the change.

**Pull request merged**